templates/bitbucket/myrror-cli.bitbucket-pipelines.yml

```diff
@@ -0,0 +1,14 @@
+pipelines:
+  default:
```

Comment:
Great work on this! Here are a few suggestions to improve the script:

1. In the first step, let's add a condition to ensure we're targeting the correct branch:
   ```sh
   if [ "${BITBUCKET_PR_DESTINATION_BRANCH}" != "${YOUR_MAIN_BRANCH}" ]; then 
       printf 'Not a target branch we want to check'; 
       exit; 
   fi
   ```

2. Please use `pull-requests:` instead of `default:` to better help the user.

These changes should make the script more robust and user-friendly.

Reply:
Bitbucket sends BITBUCKET_PR_DESTINATION_BRANCH in pull request step, (merge), but I don't get it when push to "future-branch" even if it in PR

Reply:
how I see pull request triggered by create/update PR, and Bitbucket doesn't think that push to branch updated PR...
so we should use default and call to bitbucket API to check that PR exists

Reply:
I'm not sure that it's the best solution, but I don't know how I can improve it 
![image](https://github.com/blindspot-security/myrror-cli/assets/57172661/263fe594-4560-4cd2-baed-a0fece29ef95)